streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -53,8 +54,8 @@
 
     /**
      * Create a {@link KStream} from the specified topics.
-     * The default {@code "auto.offset.reset"} strategy and default key and value deserializers as specified in the
-     * {@link StreamsConfig config} are used.
+     * The default {@code "auto.offset.reset"} strategy, default {@link TimestampExtractor} and default key and value
```

Comment:
Nit: "default {@link TimestampExtractor}[,] and"

The rule is "A and B" (for two things no comma), but "A, B, C, and D" (for three or more things, use commas)